ft-email-auth-provider/src/handlers/confirm_email.rs

```diff
@@ -0,0 +1,92 @@
+use crate::handlers::resend_confirmation_email::{
+    generate_new_confirmation_key, send_confirmation_email,
+};
+use validator::ValidateEmail;
```

Comment:
Don't use `use`.

Reply:
done in https://github.com/fastn-community/auth/pull/4/commits/e90c88048b1497f6cf86b0024118f1e49cd31768